jhub_apps/service/security.py

```diff
@@ -15,9 +15,11 @@
 ### Authorization: bearer token (header).
 ### Hub technically supports cookie auth too, but it is deprecated so
 ### not being included here.
+JHUB_APPS_AUTH_COOKIE_NAME = "jhub_apps_access_token"
+
 auth_by_param = APIKeyQuery(name="token", auto_error=False)
 
-auth_by_cookie = APIKeyCookie(name="access_token")
+auth_by_cookie = APIKeyCookie(name=JHUB_APPS_AUTH_COOKIE_NAME)
```

Comment:
What do you think about:

```suggestion
auth_by_cookie = APIKeyCookie(name=JHUB_APPS_AUTH_COOKIE_NAME)
auth_by_cookie_deprecated = APIKeyCookie(name="access_token")   # will be removed in next version
```

And adding `or auth_by_cookie_deprecated` in line 48?

Reply:
In general I'm in favor of not disrupting user workflow. However this would only shift the disruption to a later date unless the user gets warned or notified in some way that they depend on deprecated behavior.

Reply:
> However this would only shift the disruption to a later date unless the user gets warned or notified in some way that they depend on deprecated behavior

the new cookie would be set once they log in using this version, and I think the cookies only have 7 days lifetime so think the proposed deprecation solution is fine; also the disruption would be minimal, probably just accepting the oauth screen which we are removing on nebari either way

Reply:
So we are only talking about the disruption of users needing to login again once? If so, I wouldn't bother with trying to read the old one. This is just minor inconvenience and for that we need to have another PR plus review.

Reply:
I agree with both of you here, but since adding this line (removing in next version) is not much annoyance and makes it backwards compatible for the few % of affected users at least for one release. I wouldn't mind adding it.